packages/account/src/Components/poo/statuses/poo-rejected/poo-rejected.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Button, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import IconMessageContent from 'Components/icon-message-content';
+
+type TPOORejected = {
+    onTryAgain: () => void;
+};
+
+const POORejected = ({ onTryAgain }: TPOORejected) => (
+    <div className='proof-of-ownership__container'>
+        <div className='proof-of-ownership__container--content'>
+            <IconMessageContent
+                icon={<Icon icon='IcPooRejected' size={128} />}
+                is_disabled_for_mobile
```

Comment:
couldnt find a prop is_disabled_for_mobile for IconMessageContent . can you please check this @sanjam-deriv

Reply:
I guess this prop is being passed as a boolean to the `IconMessageContent` component

Reply:
yes, this is an optional boolean prop 🦴